src/components/ChainSelectionPopover/ChainSelectionPopover.styles.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components/macro";
+
+import breakPoints from "../../style/breakpoints";
+import { BorderlessButtonStyle } from "../../style/mixins";
+
+/**
+ * @remarks props are used to manipulate the positioning of components
+ */
+type ContainerProps = {
+  open: boolean;
+  shiftLeft: boolean;
+};
+
+export const Container = styled.div<ContainerProps>`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 16rem;
+  height: 14rem;
+  top: 5rem;
+  right: 3.75rem;
+  padding-top: 0.5rem;
+  transform: ${(props) => (props.shiftLeft ? "translate(-16.75rem, 0)" : "0")};
+  color: ${(props) => props.theme.colors.darkSubText};
+  background-color: ${({ theme }) =>
+    theme.name === "dark" ? theme.colors.darkGrey : theme.colors.black};
+  border: ${(props) => props.theme.colors.borderGrey} 1px solid;
+  border-radius: 0.5rem;
+  z-index: 1000;
+
+  @media ${breakPoints.tabletPortraitUp} {
+    right: 12rem;
+  }
+
+  @media ${breakPoints.tabletPortraitUpMax} {
+    right: 10rem;
+  }
+
+  @media ${breakPoints.phoneOnly} {
+    top: 4rem;
+    right: 4rem;
+  }
+`;
+
+type ChainSelectionContainerType = {
+  $overflow: boolean;
+};
+
+export const NetworksContainer = styled.div<ChainSelectionContainerType>`
+  margin: 0.5rem 0 0 0;
+  display: flex;
+  flex-direction: column;
+  height: 2.5rem;
+`;
+
+type ButtonStyleProps = {
+  $isActive: boolean;
+};
+
+export const NetworkButton = styled.button<ButtonStyleProps>`
+  ${BorderlessButtonStyle};
+
+  display: flex;
```

Comment:
You need a solution for long names:

![Screenshot 2023-06-03 at 13 27 22](https://github.com/airswap/airswap-web/assets/86911296/cef01f26-32b1-474e-ac6c-7962bacbb6dc)

Reply:
This doesn't happen on my end
![Screenshot 2023-06-03 at 8 41 43 AM](https://github.com/airswap/airswap-web/assets/90465420/a93deb5d-652f-466c-a599-dc725db367a1)

Reply:
Just trying to say the styling is not solid.

Reply:
Now I'm getting this error `MetaMask - RPC Error: Expected 0x-prefixed, unpadded, non-zero hexadecimal string 'chainId'. Received` and cannot figure out a solution to the issue. Trying for the past couple hours. I'm going to leave it as is.

As for the styles not being solid, I don't know how to replicate the error on my end, so I don't know how to address your comment

Reply:
Fixed the error `MetaMask - RPC Error: Expected 0x-prefixed, unpadded, non-zero hexadecimal string 'chainId'.`